src/iface/ethernet.rs

```diff
@@ -851,7 +851,7 @@ impl<'b, 'c, 'e> InterfaceInner<'b, 'c, 'e> {
                 // The packet is valid and handled by socket.
                 Ok(()) => handled_by_raw_socket = true,
                 // The socket buffer is full or the packet was truncated
-                Err(Error::Exhausted) | Err(Error::Truncated) => (),
+                Err(Error::Exhausted) | Err(Error::Truncated) => net_debug!("Raw packet dropped"),
```

Comment:
This change isn't related to adding capacity methods.

Reply:
Apologies, I have removed it.